examples/kokkos-tutorials/workload/bfs_bottomup.py

```diff
@@ -0,0 +1,170 @@
+from typing import Tuple
+
+import pykokkos as pk
+
+from parse_args import parse_args
+
+@pk.workload
+class Workload:
+    def __init__(self, N: int, M: int):
+        self.N: int = N
+        self.M: int = M
+
+        self.val: pk.View1D[pk.double]       = pk.View([N*M], pk.double)
+        self.visited: pk.View1D[int]         = pk.View([N*M], int)
+        self.mat: pk.View2D[pk.double]       = pk.View([N, M], pk.double)
+        self.max_arr: pk.View1D[pk.double]   = pk.View([N], pk.double)
+        self.max_arr2D: pk.View2D[pk.double] = pk.View([N, N], pk.double)
+
+        self.timer_result: float = 0
+        self.element: float = N*M
+
+        self.val.fill(N+M)
+
+        self.visited.fill(0)
+
+        # Initialize the input matrix, can be design to be any binary matrix
+        # In this example, mat[0][1] & mat[0][3] will be 0, others will be 1
+        self.mat.fill(1)
+        self.mat[0][1] = 0
+        self.mat[0][3] = 0
+
+    @pk.main
+    def run(self):
+        timer = pk.Timer()
+        self.timer_result = timer.seconds()
```

Comment:
At the moment, this timer isn't really timing anything. You can either remove it or move the call to `timer.seconds()`.